engine/script/src/luacjson/lua_cjson.c

```diff
@@ -459,7 +459,7 @@ static void json_create_config(lua_State *l)
 #endif // DEFOLD
 
 // DEFOLD
-static void json_initialize_config(json_config_t* cfg)
+static void json_initialize_config(json_config_t* cfg, int encode_keep_buffer)
```

Comment:
Here I add an ability to avoid `cfg->encode_buf` initialization in case we don't need it (when decode)